src/components/ClientList.jsx

```diff
@@ -1,8 +1,8 @@
-import React, {useEffect, useState} from 'react';
```

Comment:
React should alway be there at the top

Reply:
https://reactnative.dev/docs/intro-react

Because JSX is included in the React library, it won’t work if you don’t have import React from 'react' at the top of your file!